update-grub2

```diff
@@ -1,9 +1,10 @@
-#!/bin/bash
+#!/bin/sh
 
-
-if [ $EUID != 0 ]; then
-    printf "%s\n" "$0: Need root to run"
+if [ "$(id -u)" != "0" ]; then
+    printf '%s\n' "$0: Need root to run"
     exit 1
 fi
 
 /usr/sbin/grub2-mkconfig -o /boot/grub2/grub.cfg
+printf '%s\n' "Installing GRUB."
+/usr/sbin/grub2-install
```

Comment:
That's wrong. Why would you do that on every kernel update as an example? And if you run it why after grub2-mkconfig?

IOW, if you want that that needs to be a function() to check for whatever changes you've made, and if detected,
throw in a warning and run grub install, ofc still before mkconfig.

However I don't get why all this is needed.

Reply:
Yes this may be indeed wrong, first i'd check if EFI Label = 'openmandriva' then regenerate EFI install with EFI Label from patch above.

Reply:
this is gone